src/style.css

```diff
@@ -1,3 +1,79 @@
+html,
 body {
+  padding: 0;
   margin: 0;
+  width: 100%;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+:root {
+  --primary-color: #00acdc;
+}
+
+header {
+  justify-content: space-between;
+  display: flex;
+  background: #fff;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.navigation-list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: stretch;
+  height: 60px;
+  list-style: none;
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's better to style all elements using `.class`. For instance, instead of `list-style: none` on the `ul` element, add a class to the `ul` and define the styles for that class.